aiidalab_widgets_base/process.py

```diff
@@ -94,7 +94,7 @@ def __init__(  # pylint: disable=too-many-arguments
         if callable(inputs_generator):
             self.inputs_generator = inputs_generator
         else:
-            raise ValueError(
+            raise TypeError(
                 "The `inputs_generator` argument must be a function that "
                 f"returns input dictionary, got {inputs_generator}"
```

Comment:
To make this consistent with other such exceptions that are in this PR.

```suggestion
                f"returns input dictionary, got {type(inputs_generator)}"
```